src/Compilers/CSharp/Test/Emit/Emit/CompilationEmitTests.cs

```diff
@@ -2122,15 +2122,16 @@ public class PublicClass
     internal event System.Action InternalEvent;
 }
 ";
+            var parseOptions = TestOptions.Regular.WithNoRefSafetyRulesAttribute();
```

Comment:
nit: consider moving this down closer to where it is used (e.g. right before `var realImage` is declared)